src/math.ts

```diff
@@ -0,0 +1,7 @@
+export function max(a: number, b: number): number {
```

Comment:
Do these do something different from `Math.max`? Fiddling in the browser console, they kinda look like the same thing

Reply:
Oh, just not aware of Math.min, Math.max, I was trying to find it in the default library and my importer couldn't

Reply:
Thanks

Reply:
Fixed in 799a51c